### PR TITLE
build(blade): update version

### PR DESCRIPTION
**Pull request by rzpcibot**
This PR was opened by the [Changesets release](https://github.com/changesets/action) GitHub action. When you're ready to do a release, you can merge this and the packages will be published to npm automatically. If you're not ready to do a release yet, that's fine, whenever you add more changesets to master, this PR will be updated.

# Releases
### Major Changes

-   cc4355a: feat(blade): added 2px spacing token

    ## Breaking changes

    -   Added 2px token, thus all spacing tokens have shifted by 1 step.

    ## Migration steps

    Shifted every spacing tokens by +1

    ```diff
    - <div style={{ margin: theme.spacing[1] }} />
    + <div style={{ margin: theme.spacing[2] }} />
    ```

### Patch Changes

-   a402ef1: feat(icons): add `RefreshLeft` icon

### CODE REVIEW

**Review by divyanshu013 — COMMENTED** [review]
The changeset is bit confusing @anuraghazra

File: packages/blade/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+
+- cc4355a: feat(blade): added 2px spacing token
+
+  ## Breaking changes
+
+  - Added 2px token, thus all spacing tokens have shifted by 1 step.
+
+  ## Migration steps
+
+  Shifted every spacing tokens by +1
```
Comment (divyanshu013) [inline packages/blade/CHANGELOG.md:15]: Do you mean?
```suggestion
  Shift every spacing token other than the first one (`0th` index) by +1
```

Comment (anuraghazra) [reply packages/blade/CHANGELOG.md]: Yes thanks

File: packages/blade/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+- cc4355a: feat(blade): added 2px spacing token
+
+  ## Breaking changes
+
+  - Added 2px token, thus all spacing tokens have shifted by 1 step.
+
+  ## Migration steps
+
+  Shift every spacing token other than the first one (`0th` index) by +1
```
Comment (divyanshu013) [inline packages/blade/CHANGELOG.md:15]: We should mention if people not using the tokens directly need to change anything or not

Comment (anuraghazra) [reply packages/blade/CHANGELOG.md]: That's i think implicit anyways. If no tokens are being used no changes needed.

Comment (divyanshu013) [reply packages/blade/CHANGELOG.md]: Wasn't very evident to me :)

File: packages/blade/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
 # @razorpay/blade
 
+## 2.0.0
+
+### Major Changes
+
+- cc4355a: feat(blade): added 2px spacing token
+
+  ## Breaking changes
+
```
Comment (divyanshu013) [inline packages/blade/CHANGELOG.md:10]:
```suggestion

> **Note**
>
> This breaking change affects you only if you're using the tokens directly somewhere. If you're only using the components then nothing needs to be updated at your end.

```

Comment (divyanshu013) [reply packages/blade/CHANGELOG.md:10]: This is pending 🤔

File: packages/blade/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
 # @razorpay/blade
 
+## 2.0.0
+
+### Major Changes
+
+- cc4355a: feat(blade): added 2px spacing token
+
+  ## Breaking changes
```
Comment (divyanshu013) [inline packages/blade/CHANGELOG.md:9]:
```suggestion
  #### Breaking changes
```

File: packages/blade/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+
+> **Note**
+>
+> This breaking change affects you only if you're using the tokens directly somewhere. If you're only using the components then nothing needs to be updated at your end.
+
+  - Added 2px token, thus all spacing tokens have shifted by 1 step.
+
+  ## Migration steps
```
Comment (divyanshu013) [inline packages/blade/CHANGELOG.md:17]:
```suggestion
  #### Migration steps
```

**Review by divyanshu013 — APPROVED** [review]

**Pull request merged**